docs/user-guide/services.md

```diff
@@ -28,23 +71,10 @@ BEAMLIME_INSTRUMENT=dummy gunicorn beamlime.services.wsgi:application
 Navigate to `http://localhost:8000` to see the dashboard.
 
 
-For a local demo, run the fake detector data producer:
-
-```sh
-python -m beamlime.services.fake_detectors --instrument dummy
-```
-
-Run the detector data histogramming and accumulation service:
-
-```sh
-python -m beamlime.services.detector_data --instrument dummy --sink png
-```
-
-Note the `--sink png` argument, which will save the detector data histograms as PNG files in the `data` directory.
-The Dashboard does not support detector data yet.
-
 ## Running the services using Docker
 
+Note: The docker is somewhat out of date and not all services are available currently.
+
```

Comment:
(Minor) I prefer this kind of more visible block: (I couldn't make a suggestion because of ``` line...)

```{note}
The docker is somewhat out of date and not all services are available currently.
```